src/cookbook/design/package-fonts.md

```diff
@@ -89,7 +89,6 @@ child: Text(
   'Using the Raleway font from the awesome_package',
   style: TextStyle(
     fontFamily: 'Raleway',
-    package: 'awesome_package',
```

Comment:
This snippet is extracted from a project. You'll need to update the related project for the snippets check to pass.